### ISSUE TITLE
VW Passat GTE: Not allowed to start climatisation without plugged in power

### ISSUE THREAD

**Issue by Mirarkitty** [tillsteinbach/WeConnect-python#71]
The system refuses to start climatisation without plugged in power unless that setting is sent in the request:

...body {'callback': 'https://mobileapi.apps.emea.vwapps.io/vehicles/callbacks/.../climatisation/start', 'targetTemperature_K': 297.15}
...
...info The process could not be started. Please make sure that the charging cable is plugged and try again.

But with the setting added,

...body {'targetTemperature_K': 297.15, 'callback': 'https://mobileapi.apps.emea.vwapps.io/vehicles/callbacks/.../climatisation/start', 'climatisationWithoutExternalPower': True}
...status successful
...body {'callback': 'https://mobileapi.apps.emea.vwapps.io/vehicles/callbacks/.../climatisation/start', 'climatisationWithoutExternalPower': True, 'targetTemperature_K': 297.15}

it works.

I added this value to settingsDict in controls.py. This should probably reflect the value from the settings instead:

```
                elif 'targetTemperature_F' in settingsDict:
                    settingsDict['targetTemperature_K'] = farenheitToKelvin(settingsDict['targetTemperature_F'])
                else:
                    settingsDict['targetTemperature_K'] = celsiusToKelvin(20.5)
+            settingsDict['climatisationWithoutExternalPower'] = True

        data = json.dumps(settingsDict)
        controlResponse = self.vehicle.weConnect.session.post(url, data=data, allow_redirects=True)
        if controlResponse.status_code != requests.codes['ok']:
            errorDict = controlResponse.json()
```

**Comment by tillsteinbach** [comment]
Hey, thank you for reporting and the excellent analysis. I will release the fix as soon as possible.

**Comment by tillsteinbach** [comment]
Should be fixed in 0.47.1 release

**Comment by Mirarkitty** [comment]
Great, thanks!
